dw-date-range-select.js

```diff
@@ -68,3 +71,6 @@ export class DwDateRangeSelect extends DwSelect {
 }
 
 customElements.define("dw-date-range-select", DwDateRangeSelect);
+
+export const valueProvider = _valueProvider;
```

Comment:
you can export directly like
 `export  * as valueProvider from "./value-provider.js";`

so update this